saleor/attribute/migrations/0031_assignedpageattributevalue_page.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.2.20 on 2023-08-24 11:25
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.db.models.signals import post_migrate
+from django.apps import apps as registry
+
+from .tasks.saleor3_16 import assign_pages_to_attribute_values_task
+
+
+def data_migration(apps, _schema_editor):
+    def on_migrations_complete(sender=None, **kwargs):
+        assign_pages_to_attribute_values_task.delay()
+
+    sender = registry.get_app_config("attribute")
+    post_migrate.connect(on_migrations_complete, weak=False, sender=sender)
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("page", "0028_add_default_page_type"),
+        ("attribute", "0030_assignedproductattributevalue_product"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="assignedpageattributevalue",
+            name="page",
+            field=models.ForeignKey(
```

Comment:
By default Django creates index when creating `ForeignKey`. We need to explictly turn off that behaviour by doing `db_index=False` in `ForeignKey` field creation. And then create index if needed. 
(and I believe we should have that index in db.)
Example can be found here: https://github.com/saleor/saleor/pull/13707/files

Note: it's not yet in documentation I will try to update it asap.

Reply:
done